test/SqlExtension.Tests/Integration Tests/IntegrationTestBase.cs

```diff
@@ -22,7 +22,12 @@ public class IntegrationTestBase : IDisposable
         /// <summary>
         /// Host process for Azure Function CLI
         /// </summary>
-        private Process FunctionHost;
+        protected Process FunctionHost;
+
+        /// <summary>
+        /// Host process for Azurite emulator for local Azure Storage, required by some trigger functions.
```

Comment:
Why is this necessary just for triggers?

Reply:
The triggers need something to bind to at function startup, and the local storage emulator provides that.

Reply:
What is this "something"? And why are trigger bindings the only one that need this? Do you have docs you can link to for this?

Reply:
https://docs.microsoft.com/en-us/azure/azure-functions/functions-develop-local

If you look at the doc for "AzureWebJobsStorage", it says 
`Contains the connection string for an Azure storage account. Required when using triggers other than HTTP. For more information, see the AzureWebJobsStorage reference.
When you have the Azure Storage Emulator installed locally and you set AzureWebJobsStorage to UseDevelopmentStorage=true...`

Reply:
Please add this link to the docs in the comment